### DISCUSSION TITLE
/posts/2023/09/28vitepress%E4%B8%AD%E5%A2%9E%E5%8A%A0twitter%E5%8D%A1%E7%89%87

### DISCUSSION THREAD

**Discussion by giscus** [Justin3go/justin3go.com#124] [Announcements]
# /posts/2023/09/28vitepress%E4%B8%AD%E5%A2%9E%E5%8A%A0twitter%E5%8D%A1%E7%89%87

坚持深耕技术领域的 T 型前端程序员, 关注独立开发与 AI，喜欢 Vuejs、Nestjs, 还会点 Python、搜索引擎、NLP、Web3、后端

https://justin3go.com/posts/2023/09/28vitepress%E4%B8%AD%E5%A2%9E%E5%8A%A0twitter%E5%8D%A1%E7%89%87

**Comment by Charles5277** [comment]
.vitepress/config.ts，点击查看详细代码
.vitepress/utils/handleHeadMeta.ts，点击查看详细代码

這兩個超連結失效了

**Reply by Justin3go** [reply]
博客重构了一版，可以看看[这个链接](https://github.com/Justin3go/justin3go.com/blob/release/docs/.vitepress/config/shared.ts#L40)里面对应的代码
